x/params/go.mod

```diff
@@ -0,0 +1,136 @@
+module cosmossdk.io/x/params
+
+go 1.21.0
+
+require (
+	cosmossdk.io/api v0.7.1
+	cosmossdk.io/core v0.12.0
+	cosmossdk.io/depinject v1.0.0-alpha.4
+	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/log v1.2.1
+	cosmossdk.io/math v1.1.2
+	cosmossdk.io/store v1.0.0-rc.0
+	github.com/cometbft/cometbft v0.38.0
+	github.com/cosmos/cosmos-db v1.0.0
+	github.com/cosmos/cosmos-proto v1.0.0-beta.3
+	github.com/cosmos/cosmos-sdk v0.47.5
```

Comment:
This should use a pseudo version from main and always use latest SDK:

https://github.com/cosmos/cosmos-sdk/blob/6f996ef3c882378fa488b538575730d5e394afb5/x/upgrade/go.mod#L15
https://github.com/cosmos/cosmos-sdk/blob/6f996ef3c882378fa488b538575730d5e394afb5/x/upgrade/go.mod#L185

Reply:
Still applicable